yarn-error.log

```diff
@@ -0,0 +1,183 @@
+Arguments: 
```

Comment:
.gitignore

Reply:
Odd, seems it's already in gitignore.

Reply:
manually removed [96b34cf](https://github.com/jbx-protocol/juice-token-resolver/pull/1/commits/96b34cf0ef9c8e31d6acfffe790f143901b36b48)